x-pack/plugins/ingest_manager/common/types/models/epm.ts

```diff
@@ -252,6 +252,7 @@ export enum IngestAssetType {
 export enum DefaultPackages {
   base = 'base',
   system = 'system',
+  endpoint = 'endpoint',
```

Comment:
Installing the endpoint package by default

Reply:
cc @mostlyjason